plugins/woocommerce-admin/client/marketplace/components/tabs/tabs.tsx

```diff
@@ -57,9 +58,15 @@ const renderTabs = ( props: TabsProps ) => {
 	for ( const tabKey in tabs ) {
 		tabContent.push(
 			<Button
-				className={ classNames( 'woocommerce-marketplace__tab-button', {
-					'is-active': tabKey === selectedTab,
-				} ) }
+				className={ classNames(
+					'woocommerce-marketplace__tab-button',
+					`woocommerce-marketplace__tab-${ lowerCase(
+						tabs[ tabKey ]?.title
```

Comment:
Is there a strong reason to use the `title` here? If we just use `tabKey`, we won't need to use `lowerCase`.

Reply:
Oh, that's a great improvement. I'll change that before merging.

Reply:
Amended for both the tab buttons and tab anchors in https://github.com/woocommerce/woocommerce/pull/39618/commits/5571456b11fa3885fcc09b88c0df12c9150dc7ae